config.md

```diff
@@ -608,6 +630,29 @@ Here is a full example `config.json` for reference.
         ],
         "mountLabel": "system_u:object_r:svirt_sandbox_file_t:s0:c715,c811"
     },
+    "solaris": {
```

Comment:
On Fri, Apr 29, 2016 at 11:19:26AM -0700, Abhijeeth Nuthan wrote:
> > +    "solaris": {
> 
> I was under the impression that we could have all platform specified
> as such in config.json . The runtime would only parse the required
> fields based on image and/or the platform section of the
> config.json.  We can refactor this however we see fit though. Need
> not necessarily be the way I suggested as above.

For previous multi-platform config discussion, see #73, #74, and #76.
The consensus was “this should be handled in a higher level” [1,2].

[1]: https://github.com/opencontainers/runtime-spec/issues/73#issuecomment-171449101
[2]: https://github.com/opencontainers/image-spec